src/styles/blocks/button.css

```
@@ -0,0 +1,20 @@
+.card__product_button_btn {
+  display: flex;
+  width: 166px;
+  height: 40px;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  padding: 12px 56px 12px 55px;
+  text-transform: uppercase;
+  border: 1px solid #00acdc;
```

```suggestion
  border: 1px solid transparent;
```
А при хавері міняй колір бордера на #00acdc 
Тести, скорі за все, фейляться саме через розбіжності з бордером

---

І таке вже пробував. Але чомусь результат той самий.